CHANGELOG.md

```diff
@@ -7,6 +7,26 @@ documentation before upgrading to a new release.
 
 Released closed milestones can be found on [GitHub](https://github.com/Icinga/icinga2/milestones?state=closed).
 
+## 2.14.5 (2025-02-06)
+
+This release fixes a regression introduced in 2.14.4 that caused the `icinga2 node setup`,
+`icinga2 node wizard`, and `icinga2 pki request` commands to fail if a certificate was
+requested from a node that has to forward the request to another node for signing.
+Additionally, it fixes a small bug in the performance data normalization and includes
+various documentation improvements.
+
+### Bug Fixes
+
+* Don't close anonymous connections before sending the response for a certificate request #10337
+* Performance data: Don't discard min/max values even if crit/warn thresholds aren’t given #10339
```

Comment:
Only two bugfixes? I'd include at least one more from https://github.com/Icinga/icinga2/pulls?q=is%3Apr+label%3A%22consider+backporting%22+is%3Aclosed+milestone%3A2.15.0

Reply:
Yes, I prefer getting a quick regression fix out for #10330 rather than spending too much time going over other PRs that don't seem too important.

> I'd include at least one more

Just so that there are three instead of two? At least that doesn't sound like you'd think any one of them is particularly important.

I've additionally included #10339 because that's a fix that I understood quickly and seems to be properly tested by the unit tests.

Reply:
> Just so that there are three instead of two?

Only two bugfixes in a non-security release look a bit too few to me.

> At least that doesn't sound like you'd think any one of them is particularly important.

(Please could you formulate guesses about what I meant but didn't write as questions in the future?)

Actually, I wanted you to have the freedom to choose. After all, different people consider different stuff more/less important.

> other PRs that don't seem too important.

* The bug fixed by #7818 is indeed annoying, but tolerate-able
* The bug fixed by #7928 could _theoretically_ be worked around via default template, but I'm not sure
* The bug fixed by #8011 however **breaks check plugins** – we're Icinga, we should enable people to use check plugins